tests/0018-cgrp_term.c

```diff
@@ -260,7 +261,8 @@ static void do_test(rd_bool_t with_queue) {
         /* Let remaining consumers run for a while to take over the now
          * lost partitions. */
 
-        if (test_consumer_group_protocol_generic() &&
+        /* Callback count can vary in KIP-848 */
+        if (test_consumer_group_protocol_classic() &&
```

Comment:
We should be using `test_consumer_group_protocol_consumer` at all the places instead of `test_consumer_group_protocol_classic`. Reason behind this is:

1. Let's say we add a new protocol in future. We want all the tests to run. We are making sure of that by using `!test_consumer_group_protocol_consumer()` here instead.
2. In `!test_consumer_group_protocol_classic()` condition, we will skip those tests with the new protocol added.